Dockerfile.icq

```diff
@@ -1,24 +1,21 @@
-FROM golang:1.17-bullseye
+FROM golang:1.17-alpine3.15 AS icq-builder
 
-RUN apt install git
+RUN apk add --upgrade git
 
 WORKDIR /src/app
-RUN git clone https://github.com/Stride-Labs/interchain-queries.git
-RUN cp -r interchain-queries/* .
-RUN rm -rf interchain-queries
-
-# copy over stride module and install it
-RUN mkdir stride_ref
-RUN mkdir stride_ref/stride
-COPY . /src/app/stride_ref/stride
-WORKDIR /src/app/stride_ref/stride
-WORKDIR /src/app
-RUN go mod download
-RUN go build
 
-RUN ln -s /src/app/interchain-queries /usr/local/bin
-RUN adduser --system --home /icq --disabled-password --disabled-login icq -U 1000
-USER icq
+RUN git clone https://github.com/Stride-Labs/interchain-queries.git \
```

Comment:
Should we add a hash or tag here?

Reply:
Yeah probably a good call! Do you think ICQ is in a pretty stable state? For awhile it seemed like there were a lot of changes to main which is why I just kept this pointed there. I think we've since added a hash to the gitmodule though so maybe I'll just use that

Reply:
Actually just noticed our submodule is pointed to main as well (as is our testnet)